src/core/env/src/main/scala/FileUtilities.scala

```diff
@@ -30,6 +30,16 @@ object FileUtilities {
     }
   }
 
+  def using[T <: AutoCloseable, U](disposable: T)(task: T => U): Try[U] = {
```

Comment:
Its right under the other using, but I can refactor it somewhere else if you see fit